### DISCUSSION TITLE
Embed query param for self-hosted

### DISCUSSION THREAD

**Discussion by jam3smay** [marcveens/react-drawio#31] [Q&A] [answered]
Hi. Maybe I'm missing something, but I don't see any way to set the embed=1 query parameter which is necessary when using a self-hosted drawio server. Without the embed query parameter, the Save and Exit buttons will never display. Instead I only get a "Share" button. 

I've verified that conditionally adding the embed parameter when baseUrl is present solves the problem. I'm happy to submit a PR for this but I wanted to check first to make sure I'm not overlooking something obvious.

**Comment by marcveens** [comment, marked as answer]
Thanks for pointing this out! I also found the `embed=1` parameter in the documentations, but everything seemed to work as expected without the parameter. 

I just released a new version with the `embed=1` parameter always included!

**Reply by jam3smay** [reply]
Thanks Marc! The component is very well done.
